pages/linux/tree.md

```diff
@@ -25,3 +25,7 @@
 - Filter the tree using a wildcard (glob) pattern:
 
 `tree -P {{*.txt}}`
+
+- Exclude a directory:
```

Comment:
Perhaps "Ignore" is better than "Exclude" because the option is `-I` and we always strive to use the word used in the flag.

Also, the documentation says that `-I` can be a pattern like `-P`. Perhaps we should clarify that rather than saying just a "directory" ?

Reply:
@agnivade Fixed with 68ccb63f989bfdbac1ed6dab93edd7338aedacd5